de.fu_berlin.inf.dpp.core/META-INF/MANIFEST.MF

```diff
@@ -136,16 +136,15 @@ Export-Package: com.strangeberry.jmdns.tools,
  org.xmlpull.mxp1,
  org.xmlpull.v1
 Bundle-ClassPath: .,
- lib/smack-3.4.1-0cec571.jar,
- lib/smackx-3.4.1-0cec571.jar,
- lib/smackx-debug-3.4.1-0cec571.jar,
- lib/log4j-1.2.15.jar,
  lib/commons-codec-1.3.jar,
  lib/commons-io-2.0.1.jar,
  lib/commons-lang-2.4.jar,
- lib/xstream-1.4-20101008.163445-1.jar,
- lib/xpp3-1.1.4c.jar,
+ lib/jmdns-3.4.1.jar,
+ lib/libidn-1.15.jar,
+ lib/log4j-1.2.15.jar,
  lib/picocontainer-2.11.2.jar,
+ lib/smack-3.4.1.jar,
+ lib/smackx-3.4.1.jar,
  lib/weupnp.jar,
- lib/jmdns-3.4.1.jar,
- lib/libidn-1.32.jar
+ lib/xpp3-1.1.4c.jar,
+ lib/xstream-1.4.10.jar
```

Comment:
Please do not sort the classpath, it defines the order in which way classes are loaded. In this case it does not matter. Imagine we have a jar file that is called patches.jar

If you would insert this in the location where it fits alphabetically you would just produce unknown issues during runtime.